resources/lib/playbackmanager.py

```diff
@@ -26,6 +26,8 @@ def log(self, msg, level=2):
         ulog(msg, name=self.__class__.__name__, level=level)
 
     def launch_up_next(self):
+        if self.player.isExternalPlayer():
```

Comment:
We would need to log when this happens otherwise it will be harder to troubleshoot issues caused by an external player.